source/includes/admin_api/opt_ins/_create.json.md

```diff
@@ -0,0 +1,110 @@
+```json
+{
+  "opt_in": {
+    "_type": "opt_in",
+    "id": 123,
+    "slug": "do-you-consent-to-us-storing-your-data",
+    "name": "Do you consent to us storing your data?",
+    "description": null,
+    "release_ids": [1234567],
+    "acceptance_count": 0,
+    "created_at": "2021-08-12T16:21:00.000+01:00",
+    "updated_at": "2021-08-12T16:21:00.000+01:00",
+    "view": "extended",
+    "translations": {
+      "en": {
+        "name": "Do you consent to us storing your data?",
+        "description": null
+      }
+    },
+    "releases": [
```

Comment:
To keep things concise, we could use an empty array `[]` instead of including a release and all it's attributes because the release attributes are documented elsewhere. Also in the JSON examples for other models it's mostly an empty array for associations like this.